.github/workflows/test.yml

```diff
@@ -57,6 +60,10 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
         with:
           components: clippy, rustfmt
+      - name: Setup Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: "rstar -> target"
```

Comment:
Our workspace is in the top-level folder and the `rstar` contains the package by the same name, so I upon reading https://github.com/Swatinem/rust-cache?tab=readme-ov-file#example-usage, I would expect the default of `. -> target` to fit our use case?

Reply:
What I meant to say is that you can remove the `with: workspaces: ..` part everywhere. The defaults should work fine.